### ISSUE TITLE
BDA Extended don't load

### ISSUE THREAD

**Issue by todeilfungo** [PapaJoesSoup/BDArmory#830]
When reporting issues please provide the following information:

[See this for more information on reporting issues for KSP Mods](http://forum.kerbalspaceprogram.com/?showtopic=83212)

[See the Original Forum Post (OP) for additional info](http://forum.kerbalspaceprogram.com/?showtopic=155014)

##### ISSUE TYPE

 - Bug Report   (Something is broken)

##### KSP and BDA Version

1.12.3

1:1.2.0.5

##### OS / ENVIRONMENT
windows 11

##### SUMMARY
the loading screen won't go over the 100mm turret of BDA Extended

##### STEPS TO REPRODUCE
i Just have BDA Extenden installed and a bunch of non-related mods

##### Log Files

the log is too long, so not even with the gist i can upload it, but i have this other "log" from ksp community fixes warning: https://gist.github.com/todeilfungo/be3661aef8e13825a3e2babbbc019ea5#file-mod-log-txt
i hope it could help

**Comment by SuicidalInsanity** [comment]
BDA-E requires BDA+ 1.6.xx. The BDAc that you have installed is massively out of date and BDA-E is not compatible with it. Either remove BDA-E or upgrade your BDA install to BDA+

**Comment by todeilfungo** [comment]
> BDA-E requires BDA+ 1.6.xx. The BDAc that you have installed is massively out of date and BDA-E is not compatible with it. Either remove BDA-E or upgrade your BDA install to BDA+

dose BDA+ have the same things of BDAc?

**Comment by SuicidalInsanity** [comment]
Yes, and more. BDA+ is the most current and up-to-date version of BDA, and continued where BDAc left off when it stopped getting updates several years ago.
The only reason to continue to use BDAc1.3.4.x is if you're doing LMP multiplayer, else there's no reason not to update to the latest version (1.6.3.0).

**Comment by todeilfungo** [comment]
> Yes, and more. BDA+ is the most current and up-to-date version of BDA, and continued where BDAc left off when it stopped getting updates several years ago. The only reason to continue to use BDAc1.3.4.x is if you're doing LMP multiplayer, else there's no reason not to update to the latest version (1.6.3.0).

dose mods that were related to bda contininued work on bda+?

**Comment by SuicidalInsanity** [comment]
Mods that add things like new guns should still work, yes. BDA is fairly backwards-compatible. Mods that have 3rd-party plugins needing BDA as a requirement will need to be recompiled, however.
